[PowerToys] Update and reorganize the PT Run page

htcfreek opened the pull request:
This PR updates the PT Run docs page for  the next release (v0.57.0), fixes some mistakes, corrects outdated information and reorganizes the page.

Linked issues:
- https://github.com/microsoft/PowerToys/issues/16630
- https://github.com/microsoft/PowerToys/issues/11863

### Merge note
**Don't merge this before the release of v0.57.0 and an approval of @crutkas or @jaimecbernardo!**

htcfreek committed:
Update PT Run docs

PRMerger20 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
fixes

PRMerger6 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
layout improvement

PRMerger4 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented:
@mattwojo 
Is there a chance to see how this page looks in on the docs website after merging?

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -105,12 +135,15 @@ If the program plugin's option "Include in global result" is not selected, be su
 `.Visual Studio Code -- C:\myFolder`
 
-## Calculator Plugin
+### Calculator Plugin
```
@crutkas 
Is the list of operations up to date?

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -28,19 +28,28 @@ PowerToys Run features include:
 - Invoke Shell Plugin using `>` (for example, `> Shell:startup` will open the Windows startup folder)
 - Do a simple calculation using calculator
+- Executing system commands
+- Getting time and date informations or coverting units
```
or **converting** units

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -28,19 +28,28 @@ PowerToys Run features include:
 | Maximum number of results | Maximum number of results shown without scrolling |
 | Clear the previous query on launch | When launched, previous searches will not be highlighted |
+| Preferred monitor position | If multiple monitors are in use, PowerToys Run can be launched on the desired monitor:<br />- Primary monitor<br />- Monitor with mouse cursor<br />- Monitor with focused window |
+| App theme | Change the color theme used by PowerToys Run|
```
Run |

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -69,12 +79,25 @@ These default activation phrases will force PowerToys Run into only targeted plu
 | `{` | Visual Studio Code previously opened workspaces, remote machines (SSH or Codespaces) and containers. This plugin is off by default. | `{powertoys` to search for workspaces that contain 'powertoys' in their paths |
 | `%%` | Unit converter only | `%% 10 ft in m` to calculate the number of meters in 10 feet |
 | `$` | Windows settings only | `$ Add/Remove Programs` to launch the Windows settings menu for managing installed programs. To list all settings of an area category, type `:` after the category name. `$ Device:` to view all available Device settings |
+| '(' | Show only time and date results | `(time and date` show the current time and date in different formats. For example, to get the calendar week for a specific date you can type `(calendar week::04/01/2022`.
```
\`(\`

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -69,12 +79,25 @@ These default activation phrases will force PowerToys Run into only targeted plu
+The PowerToys Run settings menu includes a plugin manager that allows you to enable/disable the various available plugins. By selecting and expanding the sections, you can customize the direct activation commands used by each plugin. In addition, you can select whether a plugin appears in global results, as well as set additional plugin options where available.
 
-## System commands
+![PowerToys Run plugin manager](../images/pt-run-plugin-manager.png)
```
PowerToys Run Plugin Manager

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -84,15 +107,22 @@ PowerToys Run enables a set of system level actions that can be executed.
 The PowerToys Run program plugin allows for program arguments to be added when launching an application. The program arguments must follow the expected format as defined by the program's command line interface.
 
+> [!NOTE]
+> To input valid search queries, the first element after the program name has to be one of the following phrases:
```
the following **possibilities**:

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -123,28 +156,73 @@ The PowerToys Run calculator plugin supports the following operations:
+
+### Settings plugin
+
+The settings plugin allows you to search for windows settings. You can search the settings by their name or by their location.
```
for **Windows** settings

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -123,28 +156,73 @@ The PowerToys Run calculator plugin supports the following operations:
+### Windows Search settings
 
-If the Windows Search plugin is not set to cover all drives, you will receive the following warning:
+If the search settings in Windows are not set to cover all drives, you will receive the following warning when using the Windows Search plugin:
```
the **indexing settings for Windows Search** are not

htcfreek commented:
ToDo: New plugin manager screenshot with symbols.

crutkas commented on hub/powertoys/run.md:
```diff
@@ -123,28 +156,73 @@ The PowerToys Run calculator plugin supports the following operations:
 | Tangent | tan( x ) |
 | Arc Tangent | arctan( x ) |
```
We should have all arcs

htcfreek committed:
fixes, update calculator table

PRMerger8 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
improve image

PRMerger5 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented:
@crutkas 
I have updated the calculator table and added a warning about the comma bug. PTAL.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
+> The calculator plugin respects your computers number format setting. Please be aware of the different decimal delimiters in different languages when searching.
+
+> [!WARNING]
+> There is a known issue that the comma sign used in some operations as delimiter between numbers gets interpreted as decimal delimiter. This happens if your number format setting in Windows is configured that you use comma decimal separator.
```
```suggestion
> There is a known issue that the comma sign used in some operations as delimiter between numbers gets interpreted as decimal delimiter. This happens if your number format setting in Windows is configured to use the comma sign as decimal separator.
```

htcfreek committed:
fix spelling in clac warning

PRMerger14 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented:
@crutkas 
I updated the calculator table and added a warning about the comma separator bug in some methods. PTAL. 

I opened the issue https://github.com/microsoft/PowerToys/issues/17300 for the comma bug.

htcfreek committed:
fix spaces

PRMerger15 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
+| Hyperbolic Sine | sinh( x ) | |
+| Hyperbolic Cosine | cosh( x ) | |
+| Hyperbolic Tangent | tanh( x ) | |
+| Hyperbolic Arc Sine | arcsinh( x ) | |
+| Hyperbolic Arc Cosine | arccosh( x ) | |
+| Hyperbolic Arc Tangent | arctanh( x ) | |
```
arsinh
arcosh
artanh

htcfreek committed:
Merge branch 'MicrosoftDocs:docs' into PtRun

PRMerger15 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
fix hyperbolic calc operations

PRMerger5 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
+| Hyperbolic Tangent | tanh( x ) | |
+| Hyperbolic Arc Sine | arsinh( x ) | |
+| Hyperbolic Arc Cosine | arcosh( x ) | |
+| Hyperbolic Arc Tangent | artanh( x ) | |
```
`artanh( x )` isn't woking at the moment. See https://github.com/microsoft/PowerToys/issues/17323 for more details.

jaimecbernardo replied:
The function expects a number between -1 and 1 as an argument. It's working as expected.

htcfreek replied:
Show I add a hint in the third column?

htcfreek replied:
> Show I add a hint in the third column?

I think the people who need this knows how to user it.

jaimecbernardo replied:
I think not. That's part of the mathematical understanding of what the function represents. It'd be like adding a note to square root saying it doesn't accept negative numbers because the square root of negative numbers doesn't exist in real numbers.

jborsecnik commented:
@mattwojo, if you approve these changes, they should be moved to the private repo for staging and validation. We can assist with that if you wish, but it may take longer. Thanks.

htcfreek commented:
@crutkas , @jaimecbernardo
What ifs the time frame for release/merge here.

jaimecbernardo commented on hub/powertoys/run.md:
```diff
@@ -28,19 +28,29 @@ PowerToys Run features include:
 - Invoke Shell Plugin using `>` (for example, `> Shell:startup` will open the Windows startup folder)
 - Do a simple calculation using calculator
+- Executing system commands
+- Getting time and date informations
```
**informations**  -> **information**

jaimecbernardo commented on hub/powertoys/run.md:
```diff
@@ -69,12 +80,25 @@ These default activation phrases will force PowerToys Run into only targeted plu
 | `{` | Visual Studio Code previously opened workspaces, remote machines (SSH or Codespaces) and containers. This plugin is off by default. | `{powertoys` to search for workspaces that contain 'powertoys' in their paths |
 | `%%` | Unit converter only | `%% 10 ft in m` to calculate the number of meters in 10 feet |
 | `$` | Windows settings only | `$ Add/Remove Programs` to launch the Windows settings menu for managing installed programs. To list all settings of an area category, type `:` after the category name. `$ Device:` to view all available Device settings |
+| `(` | Show only time and date results | `(time and date` show the current time and date in different formats. For example, to get the calendar week for a specific date you can type `(calendar week::04/01/2022`.
```
**Show only time and date results** -> **Time and date only** to keep inline with the other entries.
**show** -> **shows**

jaimecbernardo commented on hub/powertoys/run.md:
```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
+| Division |  a / b | |
+| Modulo/Remainder | a % b | |
+| Exponentiation | a ^ b | |
+| Ceil | ceil( x.y ) |  Rounds a number up to the next largest integer. |
```
**largest**->**larger**

jaimecbernardo commented on hub/powertoys/run.md:
```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
+| Modulo/Remainder | a % b | |
+| Exponentiation | a ^ b | |
+| Ceil | ceil( x.y ) |  Rounds a number up to the next largest integer. |
+| Floor | floor( x.y ) | Rounds a number down to the next smallest integer. |
```
**smaller**

jaimecbernardo commented on hub/powertoys/run.md:
```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
+> There are some limitations for the "kill process" feature:
+> - Killing the Explorer process is only allowed, if each folder window is running in its own process.
+> - You can only kill elevated processes, if you have admin permissions (UAC).
+> - Windows of UWP apps don't know their process, until they are searched in non-minimized state.
```
nit: remove the commas from the last 3 sentences.

jaimecbernardo commented on hub/powertoys/run.md:
```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
+You can turn off the information in the PowerToys Run plugin manager options for Window Walker, or select the information to change the File Explorer settings. After selecting the information, the "Folder options" window will open.
 
-![PowerToys Run Monitor Selection](../images/pt-run-monitor.png)
+On the "Folder options" window, you can enabled the setting "Launch folder windows in a separate process".
```
**enabled**->**enable**

jaimecbernardo reviewed (changes requested):
Left some nits.

jaimecbernardo commented:
> @crutkas , @jaimecbernardo What ifs the time frame for release/merge here?

In principle, the last commits for the release have already been decided and are already merged. We're testing release candidates right now.

htcfreek committed:
resolve feedback

PRMerger13 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented:
@jaimecbernardo , @crutkas 
Nits are resolved. Should be good to go now.

jaimecbernardo reviewed (approved):
LGTM!

htcfreek committed:
update keyword list

PRMerger20 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
improvements

PRMerger14 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented:
@jaimecbernardo 
Sorry. But I didn't saw that there were more plugins missing on the keyword table.

htcfreek committed:
fixes

PRMerger8 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
Info adjustments

PRMerger18 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
fix table

PRMerger18 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek committed:
layout adjustments and requirements list update

htcfreek committed:
small fixes

PRMerger9 commented:
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented:
@jaimecbernardo 
System requirements updated.

htcfreek commented:
Who can merge this? New version of PT is released.

mattwojo commented:
> @mattwojo Is there a way to see how this page looks in on the docs website after merging?

You would need to submit it to our private repo and have access to our internal-only review site.. so unfortunately no, unless I check it for you and share a screenshot... if there is ever anything with questionable formatting, happy to do so!

mattwojo commented:
These updates look great to me. I'll merge and then confirm formatting looks good once it builds live.

htcfreek commented:
thank you for merging
